### PR TITLE
Standardize rule names

### PR DESCRIPTION
**Pull request by jkamdjou**
- Standardize rule names
- standardize
- Standardize more

### CODE REVIEW

File: detection-rules/attachment_invoice_fake_customer_service_freemail_sender.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: "Attachment invoice or receipt from freemail sender with fraudulent customer service number"
+name: "Attachment invoice or receipt from freemail sender with customer service number"
```
Comment (ianthiel) [inline detection-rules/attachment_invoice_fake_customer_service_freemail_sender.yml:1]: Should this be "Attachment: Invoice or receipt from freemail sender with customer service number" ?

File: detection-rules/open_redirect_hhs.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-name: "HHS open redirect"
+name: "Open redirect: HHS"
 description: |
   Looks for use of teh HHS open redirect.
```
Comment (ianthiel) [inline detection-rules/open_redirect_hhs.yml:3]: Just noticed there's a typo here: teh -> the

**Review by ianthiel — APPROVED** [review]
Two notes, neither is blocking

**Comment by jkamdjou** [comment]
> Two notes, neither is blocking

updated!

**Review by ianthiel — APPROVED** [review]

**Pull request merged**